fs/dir_wrapper.go

```diff
@@ -55,6 +66,10 @@ func (d *DirWrapper) Metadata(ctx context.Context) (Metadata, error) {
 func (d *DirWrapper) SetMetadata(ctx context.Context, metadata Metadata) error {
 	do, ok := d.Directory.(SetMetadataer)
 	if !ok {
+		if d.failSilently {
+			Debugf(d, "can't SetMetadata for this directory (%T from %v) -- skipping", d.Directory, d.Fs())
```

Comment:
Log message should start with a capital!

Reply:
Funny -- the linter has admonished me in the past for using capitals (and punctuation) so I think I had it in my head that this was a general style rule, but maybe that's only for errors, since they sometimes get wrapped?

As an aside, is it possible to add custom linter rules for stuff like this? Could be convenient to get that little reminder to fix it myself, and spare you the trouble 🙂

Reply:
Looking through the code, of course there are zillions of logs which don't start with a capital! Maybe it is more nuanced than I said and more a matter of taste...

```
$ git grep -P 'fs.(Debug|Info|Notice|Error)f\(.*?, "[a-z]' | wc -l
691
$ git grep -P 'fs.(Debug|Info|Notice|Error)f\(.*?, "[A-Z]' | wc -l
1388
```

The [linter has lots of config](https://golangci-lint.run/usage/configuration/) but give the 2:3 ratio above I think we should leave alone!